tasks/eth/008-fp-upgrade/input.json

```diff
@@ -0,0 +1,135 @@
+        "_implementation": "0xd81f43eDBCAcb4c29a9bA38a13Ee5d79278270cC"
+      }
+    },
+    {
+      "metadata": {
+        "name": "Reset l2Sender in OptimismPortalProxy",
+        "description": "Pre-initialization of the OptimismPortal2"
+      },
+      "to": "0xbeb5fc579115071764c7423a4f12edde41f106ed",
+      "value": "0x0",
+      "data": "0xca446dd900000000000000000000000000000000000000000000000000000000000000320000000000000000000000000000000000000000000000000000000000000000",
+      "contractMethod": {
+        "type": "function",
+        "name": "setAddress",
+        "inputs": [
+          {
+            "name": "_slot",
+            "type": "bytes32"
+          },
+          {
+            "name": "_address",
+            "type": "address"
+          }
+        ],
+        "outputs": [],
+        "stateMutability": "nonpayable"
+      },
+      "contractInputsValues": {
+        "_address": "0x0000000000000000000000000000000000000000",
+        "_slot": "0x0000000000000000000000000000000000000000000000000000000000000032"
+      }
+    },
+    {
+      "metadata": {
+        "name": "Upgrade the OptimismPortal",
+        "description": "Upgrade and initialize the OptimismPortal to OptimismPortal2 (3.10.0)"
+      },
+      "to": "0x543ba4aadbab8f9025686bd03993043599c6fb04",
+      "value": "0x0",
+      "data": "0x9623609d000000000000000000000000beb5fc579115071764c7423a4f12edde41f106ed000000000000000000000000e2f826324b2faf99e513d16d266c3f80ae87832b000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000848e819e54000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9000000000000000000000000229047fed2591dbec1ef1118d64f7af3db9eb29000000000000000000000000095703e0982140d16f8eba6d158fccede42f04a4c000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "contractMethod": {
+        "type": "function",
+        "name": "upgradeAndCall",
+        "inputs": [
+          {
+            "name": "_proxy",
+            "type": "address"
+          },
+          {
+            "name": "_implementation",
+            "type": "address"
+          },
+          {
+            "name": "_data",
+            "type": "bytes"
+          }
+        ],
+        "outputs": [],
+        "stateMutability": "nonpayable"
+      },
+      "contractInputsValues": {
+        "_data": "0x8e819e54000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9000000000000000000000000229047fed2591dbec1ef1118d64f7af3db9eb29000000000000000000000000095703e0982140d16f8eba6d158fccede42f04a4c0000000000000000000000000000000000000000000000000000000000000000",
+        "_proxy": "0xbEb5Fc579115071764c7423A4f12eDde41f106Ed",
+        "_implementation": "0xe2F826324b2faf99E513D16D266c3F80aE87832B"
+      }
+    },
+    {
+      "metadata": {
+        "name": "Upgrade SystemConfig",
```

Comment:
We'll need to clear the legacy L2OO slot, and manually set the new DGF slot. With just this, we have `address(0)` in the DGF getter

Reply:
Good catch! Fixed